ui/imports/utils/Constants.qml

```diff
@@ -897,6 +897,7 @@ QtObject {
         readonly property string mainnet: "eth"
         readonly property string arbitrum: "arb"
         readonly property string optimism: "opt"
+        readonly property string coingecko: "opt"
```

Comment:
Same than _optimism_ and is it a _network_? :thinking: 
Where/when is it used?

Reply:
This is used to build the coingecko URL for assets. Out of scope for now, but we'll need to support this in 2.29.

I'll add a task to extend `Utils.getUrlForAddressOnNetwork` to support coingeko URLs as well.

Reply:
I think the naming and value here dont match. "opt" cannnot be equated to coingecko which is a data provider for assets and opt is a network.

Reply:
removed for now. Will be implemented once we'll have the assets